src/Features/CSharp/Portable/Completion/CompletionProviders/ExplicitInterfaceMemberCompletionProvider.cs

```diff
@@ -113,6 +120,16 @@ public override async Task ProvideCompletionsAsync(CompletionContext context)
             }
         }
 
+        private static bool IsInvalidCompletionMethodKind(ISymbol symbol)
+        {
+            if (symbol is IMethodSymbol)
+            {
+                return s_invalidCodeCompletionMethodKind.Contains(((IMethodSymbol)symbol).MethodKind);
```

Comment:
Is it just easier writing `methodKind == MethodKind.PropertyGet || methodKind == methodKind.PropertySet`, etc.? I agree it's not quite as elegant, but avoid needing the field and means the function is nice and self contained.

Reply:
this can be simplified as:

```c#
            => symbol is IMethodSymbol methodSymbol && s_invalidCodeCompletionMethodKind.Contains(methodSymbol.MethodKind);
```
                return s_invalidCodeCompletionMethodKind.Contains(((IMethodSymbol)symbol).MethodKind);
```